frontend/src/components/worksheets/ExtraWorksheetHTML/ExtraWorksheetHTML.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Alert, Modal } from 'react-bootstrap';
 import './ExtraWorksheetHTML.scss';
 
-const GlossaryModal = ({ show, toggle }) => (
+const KeyboardShortCutModal = ({ show, toggle }) => (
```

Comment:
Prefer "Shortcut" over "ShortCut"

```suggestion
const KeyboardShortcutModal = ({ show, toggle }) => (
```